go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/go-kit/kit v0.10.0
 	github.com/go-logr/logr v0.2.1-0.20200730175230-ee2de8da5be6
 	github.com/go-redis/redis v6.15.9+incompatible
-	github.com/goharbor/harbor-cluster-operator v0.5.0
+	github.com/goharbor/harbor-cluster-operator v0.5.0 // indirect
```

Comment:
why we still need this pkg dependency? It can be removed, right?

Reply:
right, removed.